pepper-apis/scripts/dsm/tests/pecgs-phase4-participant.sh

```diff
@@ -3,11 +3,11 @@
 
 if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ] || [ -z "$4" ] || [ -z "$5" ] || [ -z "$6" ] || [ -z "$7" ] || [ -z "$8" ]; then
   echo "Please provide the project name and bucket name as arguments."
-  echo "Usage: ./pecgs-phase4-participant.sh <dsm-url> <participant-guid> <participant-shortId> <study> <env> <userId> <Auth0 Token file path> <BSP Token file path>"
+  echo "Usage: ./pecgs-phase4-participant.sh <dsm-url> <study> <env> <userId> <Auth0 Token file path> <BSP Token file path> <participant-guid> <participant-shortId> "
```

Comment:
The change looks fine, but it would be helpful if this script started with a comment explaining what it is used for. Without that it is hard to understand its context when just browsing the pepper code.

Reply:
I added the text that was suggested by Elliot! Thank you both

Reply:
@Rinchan20 - heads up!  This will change script parameter ordering!